lighthouse-core/test/config/config-test.js

```diff
@@ -1021,7 +1021,10 @@ describe('Config', () => {
       const extendedConfig = new Config(extended);
 
       assert.equal(config.passes.length, 1, 'did not filter config');
-      assert.deepStrictEqual(config, extendedConfig, 'had mutations');
+      // Clone so we do not compare event callbacks binded on gatherers.
+      assert.deepStrictEqual(
+        JSON.parse(JSON.stringify(config)),
```

Comment:
this wipes out a bunch of checks on these objects, so i'm a bit hesitant to loosen things this much..

whats the motivation?

Reply:
the source map gatherer sets a .bind property in its ctor (so two instances are always gonna be not equal). i think this test broke when I had source maps in the default config ... but the plan has changed since so i'll revert this change. Will prop up again when we move maps to default config..

Reply:
We ran into this again in https://github.com/GoogleChrome/lighthouse/pull/10623

The `should generate the same filtered config, extended or original` test in `config-test.js` will break if there's any `.bind()` property in the ctor (like @connorjclark said).  This test filters down to ` onlyCategories: ['performance'],`.. and the performance category, previous to 10090 and 10623, had ZERO gatherers with bound instance props. (Just by coincidence).  Whereas best-practices and other categories definitely have them.  

So this was a fun lil unexpected failure. 

In #10623, @Beytoven is adding some code to delete instances before doing the diff which will sort this out.